src/components/views/rooms/wysiwyg_composer/hooks/useWysiwygSendActionHandler.ts

```diff
@@ -46,6 +46,7 @@ export function useWysiwygSendActionHandler(
 
             switch (payload.action) {
                 case "reply_to_event":
+                case Action.FocusAComposer:
```

Comment:
This is the action that is fired when a user enters a bad command and then makes a decision from the dialog they're presented with - it ensures that we put the cursor back in the composer when the dialog closes.

Reply:
Is that documented in the actions file? Would be good to if not